app/domain/authentication/authn_jwt/signing_key/create_signing_key_provider.rb

```diff
@@ -54,13 +41,23 @@ def neither_sigining_keys_providers_configured
         def provider_uri_has_valid_configuration?
```

Comment:
what does it mean "has_valid_configuration"?
 the method only checks if it exists

Reply:
renamed to `provider_uri_resource_exists?`